### PR TITLE
nextMillennium Bid Adapter : added gpid support

### PR DESCRIPTION
**Pull request by JacobKlein26**
## Type of change

- [ ] Feature
## Description of change
Started passing gpid and pbadslot

### CODE REVIEW

**Review by patmmccann — APPROVED** [review]

File: test/spec/modules/nextMillenniumBidAdapter_spec.js [hunk 1]
```diff
@@ -100,6 +100,72 @@ describe('nextMillenniumBidAdapterTests', () => {
...
+        expected: {
+          id: 'e36ea395f67a',
+          ext: {
+            prebid: {storedrequest: {id: '123'}},
+            gpid: 'imp-gpid-123'
+          },
+          banner: {w: 300, h: 250, format: [{w: 300, h: 250}, {w: 320, h: 250}]},
+        },
+      },
+
```
Comment (patmmccann) [inline test/spec/modules/nextMillenniumBidAdapter_spec.js:132]: linting error on this line

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/nextMillenniumBidAdapter.js [hunk 2]
```diff
@@ -291,6 +291,11 @@ export function getImp(bid, id, mediaTypes) {
     },
   };
 
+  const gpid = bid?.ortb2Imp?.ext?.gpid;
+  const pbadslot = bid?.ortb2Imp?.ext?.data?.pbadslot;
+  if (gpid) imp.ext.gpid = gpid;
+  if (pbadslot) imp.ext.data = { pbadslot };
```
Comment (patmmccann) [inline modules/nextMillenniumBidAdapter.js:297]: i recommend if(pbadslot && pbadslot !===gpid) or something

Comment (JacobKlein26) [reply modules/nextMillenniumBidAdapter.js:297]: If it's the same, why shouldn't we just pass both?

File: test/spec/modules/nextMillenniumBidAdapter_spec.js (hunk 1 above)
Comment (JacobKlein26) [reply test/spec/modules/nextMillenniumBidAdapter_spec.js]: @patmmccann Thanks!

I fixed it

**Comment by JacobKlein26** [comment]
Hi @patmmccann 

If it's possible to get this in to tomorrows release I would greatly appreciate it!

File: modules/nextMillenniumBidAdapter.js (hunk 2 above)
Comment (patmmccann) [reply modules/nextMillenniumBidAdapter.js:297]: Save a penguin

Comment (patmmccann) [reply modules/nextMillenniumBidAdapter.js:297]: Up to you though

**Review by patmmccann — APPROVED** [review]

**Pull request merged**